pkgs/development/libraries/freeimage/default.nix

```
@@ -1,48 +1,40 @@
-{ lib, stdenv, fetchurl, unzip, darwin }:
-
-# TODO: consider unvendoring various dependencies (libpng, libjpeg,
-# libwebp, zlib, ...)
+{ lib, stdenv, fetchsvn, darwin, libtiff
+, libpng, zlib, libwebp, libraw, openexr, openjpeg
+, libjpeg, jxrlib, pkgconfig, fetchpatch }:
```

```suggestion
, libjpeg, jxrlib, pkg-config }:
```

The `pkgconfig` is an alias (abac6afe93a60c6078c7d86b93fbbace4da3b363) that we should be looking to remove.

---

Still need to kill `, fetchpatch`